book/website/collaboration/research-infrastructure-roles/data-steward.md

```diff
@@ -4,6 +4,16 @@
 Data Stewards are a growing role within scientific communities.
 'Data Steward' is an umbrella term for numerous support roles that involve the creation, management and usage of research data (see also the {ref}`rr-rdm` chapter).
 
+```{figure} ../../figures/data-stewards.*
+---
+height: 400px
+name: data-stewards
+alt: Cartoon-like sketch of two characters depicted as data stewards in superhero attire, with the left figure gesturing towards symbols of knowledge and alert, and the right figure pointing to tools of the trade such as security and connectivity, all encompassed by a theme of data management and protection.
```

Comment:
I would add in a description of the colour scheme

Reply:
Thank you @Arielle-Bennett!